backend/common/src/test/java/ai/verta/modeldb/common/nats/JetstreamConnectorTest.java

```diff
@@ -0,0 +1,158 @@
+package ai.verta.modeldb.common.nats;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import io.nats.client.*;
+import io.nats.client.api.ConsumerConfiguration;
+import io.nats.client.api.ConsumerInfo;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import lombok.extern.log4j.Log4j2;
+import np.com.madanpokharel.embed.nats.EmbeddedNatsConfig;
+import np.com.madanpokharel.embed.nats.EmbeddedNatsServer;
+import np.com.madanpokharel.embed.nats.NatsServerConfig;
+import np.com.madanpokharel.embed.nats.NatsVersion;
+import org.junit.jupiter.api.*;
+
+@Log4j2
+class JetstreamConnectorTest {
+  public static EmbeddedNatsServer embeddedNatsServer;
```

Comment:
This is awesome!

Reply:
Wow I just saw that you are a contributor of this project! That's even more awesome!

Reply:
what? I don't think I'm a contributor unless I missed something!

Reply:
oh! haha. https://github.com/madansp/nats-embedded/pull/6 That's my only contribution.